src/routes/settings/feedback.ts

```diff
@@ -0,0 +1,29 @@
+import Joi from 'joi';
+import { ControllerGeneratorOptionsWithoutClientOrSupplier } from '../../comms/controller';
+
+interface Request {
+  body: {
+    theme: string;
+    feedback: string;
+  };
+}
+
+type Response = void;
+
+const controllerGeneratorOptions: ControllerGeneratorOptionsWithoutClientOrSupplier = {
+  method: 'post',
+  path: '/',
+  right: {},
+  body: Joi.object().keys({
+    theme: Joi.string().example('Email theme'),
```

Comment:
**question**

I don't see anything wrong with this, but why not make the theme mandatory too ?